components/canvas.py

```diff
@@ -1,20 +1,26 @@
 from OpenGL.GL import *
 from OpenGL.GLU import *
 import wx
-from wx import glcanvas
+from wx.glcanvas import GLCanvas
+from wx.glcanvas import GLContext
```

Comment:
This is minor thing. To make these two lines into one line, you can import two classes through one 'from' statement by separating  them by comma.